modules/ui/fb_feature_picker.js

```diff
@@ -149,6 +149,60 @@ export function uiFbFeaturePicker(context, keybinding) {
   }
 
 
+  function previewTags(selection, tagsObj) {
+
+    if (!tagsObj) return;
+
+    let tagPreview = selection
+      .append('div');
+
+    tagPreview
+      .attr('class', 'tag-preview'); 
+
+    var tagEntries= [];
+    Object.keys(tagsObj).forEach(k => { tagEntries.push({'key':k, 'value':tagsObj[k]});});
```

Comment:
This is an extremely minor nitpick, but I'd write this as a `map` returning Object literals instead of a `forEach` that builds up the array up by pushing to it..

My reasoning is that `map` sends the signal to anyone reading the code that you're iterating over an array and returning a transformed array.  Also it's _possible_ the JavaScript interpreter may be able optimize it better, but who really knows.

(Aside: modern JavaScript has [`Object.entries()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) which does this even more efficiently, but we don't have it polyfilled in iD yet, so we can't use it.  There's also a [`d3.entries()`](https://github.com/d3/d3-collection#entries) that does the same thing, but I feel kind of silly using the d3 helpers now that JavaScript has these features).

```suggestion
    const tagEntries = Object.keys(tagsObj).map(k => ({ key: k, value: tagsObj[k] }));
```

Reply:
I wouldn't call this extremely minor- you're 100% correct that 'forEach' carries less semantic weight than a 'map()' call, and code readability is definitely improved from your suggestion!

Reply:
Fixed in the latest commit. Woo!